src/sushi.sol

```diff
@@ -85,23 +93,27 @@ contract SushiJoin is CropJoin {
         address bonus_,
         address masterchef_,
         uint256 pid_,
-        address initialMigrator_,
-        address timelockOwner_
+        address migrator_,
```

Comment:
`migrator_`, `rewarder_`, `timelock_`, and `bonus_`, and `gem_` constructor params are all unnecessary here, as they can be asserted via their public variables after deployment in test and before governance acceptance.

Reply:
I mean why not do both? I like having a bunch of sanity checks in case something changes especially with code this heavily dependant on third-party dependencies. Constructor gas is a one-time thing, so I don't see the harm in doing some extra checks for safety.